packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -1291,4 +1290,46 @@ export class DashboardModel {
             .distinctOn(`${DashboardVersionsTableName}.dashboard_id`)
             .where(`${ProjectTableName}.project_uuid`, projectUuid);
     }
+
+    async moveToSpace(
+        {
+            projectUuid,
+            itemUuid: dashboardUuid,
+            newParentSpaceUuid,
+        }: {
+            projectUuid: string;
+            itemUuid: string;
+            newParentSpaceUuid: string;
+        },
+        {
+            transaction = this.database,
+        }: {
+            transaction?: Knex;
+        } = {
+            transaction: this.database,
+        },
```

Comment:
empty object as a default would be the same


```suggestion
        {
            transaction = this.database,
        }: {
            transaction?: Knex;
        } = {},
```